lib/Sema/CSBindings.cpp

```diff
@@ -1484,6 +1484,23 @@ PotentialBindings::inferFromRelational(Constraint *constraint) {
         AdjacentVars.insert({typeVar, constraint});
     }
 
+    // Infer a binding from `inout $T <convertible to> Unsafe*Pointer<...>?`.
+    if (first->is<InOutType>() &&
+        first->getInOutObjectType()->isEqual(TypeVar)) {
+      if (auto pointerTy = second->lookThroughAllOptionalTypes()
+                               ->getAnyPointerElementType()) {
```

Comment:
Nit: This should probably be named `pointeeType`

Reply:
✅